hspec-core/src/Test/Hspec/FailureReport.hs

```diff
@@ -25,8 +32,12 @@ writeFailureReport x = do
   where
     onError err = do
       hPutStrLn stderr ("WARNING: Could not write environment variable HSPEC_FAILURES (" ++ show err ++ ")")
+#endif
 
 readFailureReport :: IO (Maybe FailureReport)
+#ifdef __GHCJS__
+readFailureReport = return Nothing
```

Comment:
Is this required? `lookupEnv` works with GHCJS, right?  So I guess this would result in `Nothing` anyway, even without the CPP.